PollTimer.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+#include <libstuff/libstuff.h>
```

Comment:
No need to include this; it's already included as the first line of every cpp file